crypto/multisig/bitarray/compact_bit_array_test.go

```diff
@@ -7,12 +7,12 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	cmn "github.com/tendermint/tendermint/libs/common"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
```

Comment:
better to be consistent I think. i.e. use 1 name everywhere - `tmrand`

Reply:
I tend to disagree because there isn't a need to alias if nothing in the import section isn't conflicting. There are places we are using `mrand` for `math/rand` and `crand` for `crypto/rand` and places we are not aliasing them.  If this isn't a good practice to follow then I can change it and do this going forward.

Reply:
okay, let's leave it then